releasenotes/notes/fix-pytest-always-create-module-971348889d0cdf36.yaml

```diff
@@ -0,0 +1,6 @@
+---
+prelude: >
+    CI Visibility: Breaking change: test.suite and test.full_name are changed, so any visualisation or monitor that uses these fields is potentially affected
```

Comment:
does this support markdown? Also, I'd lead with breaking change
```suggestion
    **Breaking change** for CI Visibility: `test.suite` and `test.full_name` are changed, so any visualisation or monitor that uses these fields is potentially affected
```

Reply:
> does this support markdown?

Not markdown, but reStructuredText, so we can bold/add the formatting you are suggesting (just different syntax).